src/shared-components/alert/style.js

```diff
@@ -0,0 +1,118 @@
+import styled, { keyframes, css } from 'react-emotion';
+
+import {
+  COLORS,
+  MEDIA_QUERIES,
+  BOX_SHADOWS,
+  SPACING,
+  ANIMATION,
+  TYPOGRAPHY_CONSTANTS,
+} from '../../constants';
+
+const fadeInDesktop = keyframes`
+  from { opacity: 0; transform: translate3d(24px, 0, 0); }
+  to { opacity: 1; transform: translate3d(0, 0, 0); }
+`;
+
+const fadeInMobile = keyframes`
+  from { opacity: 0; transform: translate3d(0, -24px, 0);  }
+  to { opacity: 1; transform: translate3d(0, 0px, 0); }
+`;
+
+const successAlertStyles = css`
+  background-color: ${COLORS.statusGreenBackground};
+  color: ${COLORS.statusGreen};
+  fill: ${COLORS.statusGreen};
+  border-width: 1px;
+  border-color: ${COLORS.statusGreenBorder};
+  border-style: solid;
+`;
+
+const errorAlertStyles = css`
+  background-color: ${COLORS.statusRedBackground};
+  color: ${COLORS.statusRed};
+  fill: ${COLORS.statusRed};
+  border-width: 1px;
+  border-color: ${COLORS.statusRedBorder};
+  border-style: solid;
+`;
+
+const defaultAlertStyles = css`
+  background-color: ${COLORS.statusGreyBackground};
+  color: ${COLORS.statusGrey};
+  fill: ${COLORS.statusGrey};
+  border-width: 1px;
+  border-color: ${COLORS.statusGreyBorder};
+  border-style: solid;
+`;
+
+export const AlertContainer = styled.div`
+  align-items: flex-start;
+  animation: ${fadeInMobile} ${ANIMATION.defaultTiming} 1;
+  cursor: pointer;
+  display: flex;
+  padding: ${SPACING.xsmall} 0px;
+  font-size: ${TYPOGRAPHY_CONSTANTS.fontSize.caption};
+  justify-content: space-around;
+  opacity: ${props => (props.exiting ? '0' : '1')};
```

Comment:
werent you talking about how we need to get away from interpolated functions like this (and the one later)

Reply:
i believe it's only on `css` functions. When it's `styled.div` it's ok to do because it is an actual React component

Reply:
👍